tests/unittests/OperatorTest.cpp

```diff
@@ -3853,6 +3853,83 @@ TEST_P(OperatorTest, GroupConvolution) {
   EXPECT_FLOAT_EQ(result.at({0, 1, 0, 5}), (13 + 14 + 15 + 16) * 100000);
 }
 
+/// Test the functionality of groupwise quantized convolution expressed using
+/// ChannelwiseQuantizedConvNode.
+TEST_P(OperatorTest, GroupwiseQuantizedConvolution) {
+  ENABLED_BACKENDS(Interpreter, CPU, OpenCL);
+
+  constexpr size_t groups = 2;
+
+  auto *input =
+      mod_.createPlaceholder(ElemKind::FloatTy, {1, 2, 1, 8}, "input", false);
+  auto IH = bindings_.allocate(input)->getHandle<float>();
+  for (size_t i = 0; i < 2 * 8; i++) {
+    IH.raw(i) = i + 1;
+  }
+
+  auto *qInTy = mod_.uniqueType(ElemKind::Int8QTy, {1, 2, 1, 8}, 1.0, 0);
+  auto *qInput = F_->createQuantize("qInput", input, qInTy);
+
+  auto filterT = Tensor(ElemKind::Int8QTy, {6, 1, 1, 4}, 1.0, 0);
+  for (size_t i = 0; i < 6; i++) {
+    for (size_t j = 0; j < 4; j++) {
+      filterT.getHandle<int8_t>().at({i, 0, 0, j}) = (i % 2) + 1;
+    }
+  }
+  auto *filter = mod_.createConstant("filter", filterT);
+
+  auto biasT = Tensor(ElemKind::FloatTy, {6});
+  biasT.zero();
+  auto *bias = mod_.createConstant("bias", biasT);
```

Comment:
nit: when we create constants like this and below, we are unnecessarily creating two copies of the Tensor, because `createConstant()` here makes a copy of the passed in tensor (e.g. `biasT`). You could instead create the Constant first with a Type, and then get the Constant's payload and modify it.

Reply:
std::moved them